test/lexer/Lexer.test.js

```diff
@@ -353,6 +353,19 @@ describe("lexer", () => {
             ]);
         });
 
+        it("reads conditional directives", () => {
```

Comment:
Can you change the name of the test to not match the above or just combine them into one string?

Reply:
Oops. copy-paste error. Fixed. (i combined them into a single test per your suggestion).